eng/pipelines/doc-index.yml

```diff
@@ -0,0 +1,60 @@
+jobs:
+  - job: "Build"
+    variables:
+      - template: templates/variables/globals.yml
+    pool:
+      vmImage: "windows-2019"
+    steps:
+      - pwsh: |
+          Invoke-WebRequest -Uri "https://github.com/dotnet/docfx/releases/download/v2.43.2/docfx.zip" `
+          -OutFile "docfx.zip" | Wait-Process; Expand-Archive -Path "docfx.zip" -DestinationPath "./docfx/"
+        workingDirectory: $(Build.BinariesDirectory)
+        displayName: Download and Extract DocFX
+
+       - pwsh: |
+          $(Build.BinariesDirectory)/docfx/docfx.exe init -q
+        displayName: Provision DocFX Directory
+        workingDirectory: $(Build.SourcesDirectory)
+  
+      - pwsh: |
+          mkdir "templates"
```

Comment:
What doe we end up putting into this directory?

Reply:
This is for docfx